src/main/java/com/pinterest/secor/parser/PartitionFinalizer.java

```diff
@@ -41,12 +39,13 @@
 public class PartitionFinalizer {
     private static final Logger LOG = LoggerFactory.getLogger(PartitionFinalizer.class);
 
-    private SecorConfig mConfig;
-    private ZookeeperConnector mZookeeperConnector;
-    private TimestampedMessageParser mMessageParser;
-    private KafkaClient mKafkaClient;
-    private QuboleClient mQuboleClient;
-    private String mFileExtension;
+    private final SecorConfig mConfig;
+    private final ZookeeperConnector mZookeeperConnector;
+    private final TimestampedMessageParser mMessageParser;
```

Comment:
Its fine for now but in the future we may want to make it easier to plug different message parsers and partitioners into the finalizer.